Classes/Security/Authorization/Privilege/Node/Doctrine/PropertyConditionGenerator.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace PunktDe\NodeRestrictions\Security\Authorization\Privilege\Node\Doctrine;
+
+/*
+ *  (c) 2017 punkt.de GmbH - Karlsruhe, Germany - http://punkt.de
+ *  All rights reserved.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Security\Authorization\Privilege\Entity\Doctrine\SQLFilter;
+use Neos\Flow\Security\Authorization\Privilege\Entity\Doctrine\PropertyConditionGenerator as NeosPropertyConditionGenerator;
+
+/**
+ * {@inheritdoc}
+ */
+class PropertyConditionGenerator extends NeosPropertyConditionGenerator
+{
+    /**
+     * @param mixed $operandDefinition
+     * @return PropertyConditionGenerator the current instance to allow for method chaining
+     */
+    public function postgresJsonContains($operandDefinition)
+    {
+        $this->operator = '@>';
+        $this->operandDefinition = $operandDefinition;
+        $this->operand = $this->getValueForOperand($operandDefinition);
+        return $this;
+    }
+
+    /**
+     * @param SQLFilter $sqlFilter
+     * @param string $propertyPointer
+     * @param string $operandDefinition
+     * @return string
+     */
+    protected function getConstraintStringForSimpleProperty(SQLFilter $sqlFilter, $propertyPointer, $operandDefinition = null)
+    {
+        $parentReturn = parent::getConstraintStringForSimpleProperty($sqlFilter, $propertyPointer, $operandDefinition);
```

Comment:
This object already has the entity manager injected. Wouldn't it be better to check explicitly for postgres instead of the '@>' sequence?